Gems/Meshlets/Assets/Passes/MeshletsParent.pass

```diff
@@ -0,0 +1,50 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "PassAsset",
+    "ClassData": {
+        "PassTemplate": {
+            "Name": "MeshletsParentPassTemplate",
+            "PassClass": "ParentPass",
+            "Slots": [
+            ],
+            "PassRequests": [
+                {
+                    "Name": "MeshletsComputePass",
+                    "TemplateName": "MeshletsComputePassTemplate",
+                    "Connections": [
+                    ],
+                    "PassData": {
+                        "$type": "ComputePassData",
+                        "ShaderAsset": {
+                            "FilePath": "Shaders/MeshletsCompute.shader"
+                        }
+                    }
+                },
+                {
+                    "Name": "MeshletsRenderPass",
+                    "TemplateName": "MeshletsRenderPassTemplate",
+                    "Connections": [
```

Comment:
no since the connection is done at the lower level of the MeshletsRender.pass.
Notice that the meshlets parent pass was not completed and connected as per the remark - the reason is because the 'LowEndPipeline' and 'MainPipeline' do not have the same passes required for the connection.
This is something on the 'to do' and mentioned as such to properly inject it rather than change the main pipeline.